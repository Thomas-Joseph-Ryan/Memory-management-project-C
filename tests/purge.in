set a 1
set b 2 3
list keys
list entries
list snapshots
pick a 0
pick b 1
get b
pluck b 2
get b
del b
get b
set b 4 5
set c b 5
set d c
purge c
list entries
purge d
list entries
snapshot
pluck c 1
list entries
purge b
drop 1
purge b
list entries
bye