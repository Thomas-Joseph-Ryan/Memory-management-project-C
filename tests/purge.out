> ok

> ok

> b
a

> b [2 3]
a [1]

> no snapshots

> index out of range

> 2

> [2 3]

> 3

> [2]

> ok

> no such key

> ok

> ok

> ok

> not permitted

> d [c]
c [b 5]
b [4 5]
a [1]

> ok

> c [b 5]
b [4 5]
a [1]

> saved as snapshot 1

> b

> c [5]
b [4 5]
a [1]

> not permitted

> ok

> ok

> c [5]
a [1]

> bye
